bots/running-a-botpress-bot/README.md

```diff
@@ -2,4 +2,128 @@
 
 Our work on supporting Botpress is in active development.
 
+## How to setup Rocket.Chat livechat + botpress 
```

Comment:
A lot of the content below covers details that exist in existing docs for creating bot users, configuring bots, administrating livechat. I would avoid reproducing that content, because it makes it harder to review and update the docs with multiple sources of truth. I'm adding a doc to the PR for "Using bots for livechat" that just points to those other sources. It would be best to update this doc to just provide references for Botpress specific setup and deployment instructions.